packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -383,6 +416,14 @@ export class VisualizerNode extends TypedEventEmitter<DataVisualizerEvents> impl
 		);
 	}
 
+	/**
+	 * Begins the process of applying an edit to a Fluid object.
```

Comment:
I get the intention behind the "Begins the process of..." phrasing, but it isn't typically how we document async code in our codebase. I would recommend just documenting what it does - the fact that it returns a promise is (in my opinion) enough signal that the action is async. If you would like, you can also add a `@returns A promise that resolves when ...` block at the end of the comment, but that's totally up to you.